### ISSUE TITLE
[*] Starting - Apimap reordering issue : comparison of Hash with Hash failed

### ISSUE THREAD

**Issue by josephpage** [ForestAdmin/forest-rails#236]
I got this error in logs, when launching Rails.

```
=> Booting Puma
=> Rails 5.1.5.rc1 application starting in development 
=> Run `rails server -h` for more startup options
[2018-02-08 16:46:00] Forest 🌳🌳🌳  An Apimap reordering issue occured: comparison of Hash with Hash failed
Puma starting in single mode...
* Version 3.11.2 (ruby 2.4.3-p205), codename: Love Song
* Min threads: 5, max threads: 5
* Environment: development
* Listening on tcp://0.0.0.0:3000
Use Ctrl-C to stop
```

## Context

I just updated forest_liana to the brand new `2.3.0`
 
* Package Version: 2.3.0
* Rails Version: 5.1.5.rc1
* Database Dialect: pg
* Database Version: 10.1

**Comment by arnaudbesnier** [comment]
Hey @josephpage, this is something we need to fix.
Don't worry about the warning, everything will work as usual.

**Comment by RemiDuvoux** [comment]
Just had this same bug in 3.0.1

=> Booting Puma
=> Rails 5.1.4 application starting in development 
=> Run `rails server -h` for more startup options
Exiting
/Users/macbook/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/forest_liana-3.0.1/lib/forest_liana/schema_file_updater.rb:110:in `sort': comparison of Hash with Hash failed (ArgumentError)
	from /Users/macbook/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/forest_liana-3.0.1/lib/forest_liana/schema_file_updater.rb:110:in `block in initialize'
	from /Users/macbook/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/forest_liana-3.0.1/lib/forest_liana/schema_file_updater.rb:109:in `map'
	from /Users/macbook/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/forest_liana-3.0.1/lib/forest_liana/schema_file_updater.rb:109:in `initialize'
	from /Users/macbook/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/forest_liana-3.0.1/lib/forest_liana/bootstraper.rb:49:in `new'
	from /Users/macbook/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/forest_liana-3.0.1/lib/forest_liana/bootstraper.rb:49:in `generate_apimap'
	from /Users/macbook/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/forest_liana-3.0.1/lib/forest_liana/bootstraper.rb:26:in `initialize'

**Comment by arnaudbesnier** [comment]
Hi @RemiDuvoux, does it prevents you server to start properly?.

**Comment by ElisaRegnier** [comment]
Hi @arnaudbesnier, same issue for me, can't start a server/console or anything using rails.
Any idea on how to fix this ? Same error, coming from:
https://github.com/ForestAdmin/forest-rails/blob/devel/lib/forest_liana/schema_file_updater.rb
Line 110
`comparison of Hash with Hash failed (ArgumentError)`

**Comment by VincentMolinie** [comment]
Fixed in  3d5fc00 ;)
